src/reporting/reportingClient.ts

```diff
@@ -298,6 +684,16 @@ export class ReportingClient {
e that you want to select. The value of `ECPropertyName` can also be a path defining how to find a property. The path segments must be separated by a period (`.`). The path can contain the names of:    - A navigation property  - A struct property  - A string property that happens to contain a string representation of a json object  - A property inside the selected json    Names of properties are not case sensitive. If json object does not have duplicate property names which only differ in letter casing, then those json properties are not case sensitive. We recommend treating json properties as case sensitive.    - If `Category` property is a navigation property with a value of `{\"id\":\"0x2000000000b\",\"relClassName\":\"BisCore.GeometricElement3dIsInCategory\"}` and the `ECPropertyName` is set to `Category` or `category`, then the whole value `{\"id\":\"0x2000000000b\",\"relClassName\":\"BisCore.GeometricElement3dIsInCategory\"}` will be selected  - If `Category` property is a navigation property with a value of `{\"id\":\"0x2000000000b\",\"relClassName\":\"BisCore.GeometricElement3dIsInCategory\"}` and the `ECPropertyName` is set to `Category.id`, then only the `id` value `0x2000000000b` will be selected  - If `Category` property is a navigation property and it points to a row that has a property `CodeValue` that we want to select, the `ECPropertyName` should be set to `Category.CodeValue`  - If `Model` property is a navigation property and it points to a row that has a property `JsonProperties` with a value `{\"formatter\":{\"mastUnit\":{\"label\":\"m\"}}}` and we want to select the master unit label, the `ECPropertyName` should be set to `Model.JsonProperties.formatter.mastUnit.label`. The result will be `m`    ### Authentication    Requires `Authorization` header with valid Bearer token for scope `insights:modify`.    For more documentation on authorization and how to get access token visit [OAUTH2 Authorization](https://developer.bentley.com/apis/overview/authorization/) page.    ### Authorization    User must have `imodels_write` permission(s) assigned at the Project level. iModel specific permissions may also be applied at the iModel level if iModel level permissions are enabled.    Alternatively the user should be an Organization Administrator for the Organization that owns a given Project or iModel.    An Organization Administrator must have at least one of the following roles assigned in User Management: Account Administrator, Co-Administrator, or CONNECT Services Administrator. For more information about User Management please visit our Bentley Communities [Licensing, Cloud, and Web Services](https://communities.bentley.com/communities/other_communities/licensing_cloud_and_web_services/w/wiki/50711/user-management-2-0) wiki page.    ### Rate limits    All iTwin Platform API operations have a rate limit. For more documentation on that visit [Rate limits and quotas](https://developer.bentley.com/apis/overview/rate-limits/) page.    ---
```

Comment:
Shall we put repeated docs in a single region like top of all mappings and point to the place at the end

Reply:
I added the JSDocs so that developers will have intellisense on the ReportingClient